app/src/main/java/org/jboss/hal/client/installer/UpdateColumn.java

```diff
@@ -158,7 +163,17 @@ public HTMLElement getIcon() {
 
             @Override
             public HTMLElement element() {
-                return ItemDisplay.withSubtitle(item.getName(), Format.mediumDateTime(item.getTimestamp()));
+                final HtmlContentBuilder<HTMLElement> builder = span()
```

Comment:
Please add the missing CSS class `itemText`
 https://github.com/hal/console/blob/3c853acc2d293826b55bcda34de2fbdb2b43fda2/core/src/main/java/org/jboss/hal/core/finder/ItemDisplay.java#L113

```java
span().css(itemText)
```